app/src/main/java/org/simple/clinic/onboarding/OnboardingScreen.kt

```diff
@@ -0,0 +1,64 @@
+package org.simple.clinic.onboarding
+
+import android.content.Context
+import android.text.style.TextAppearanceSpan
+import android.util.AttributeSet
+import androidx.constraintlayout.widget.ConstraintLayout
+import kotlinx.android.synthetic.main.screen_onboarding.view.*
+import org.simple.clinic.R
+import org.simple.clinic.util.Truss
+
+class OnboardingScreen(context: Context, attributeSet: AttributeSet) : ConstraintLayout(context, attributeSet) {
+
+  override fun onFinishInflate() {
+    super.onFinishInflate()
+    if (isInEditMode) {
+      return
+    }
+    setIntroTextViews()
+  }
+
+  private fun setIntroTextViews() {
+    val introOneFormattedString = Truss()
+        .pushSpan(TextAppearanceSpan(context, R.style.Clinic_V2_TextAppearance_Subtitle1Left_Grey1))
```

Comment:
Perhaps we can only push the emphasis span let the rest of them use default text view style?

Reply:
This doesn't seem to be addressed in this commit